data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1441,10 +1441,10 @@ Marksman attacks are only affected if the chance to hit is greater than 60%."
                     name = opportunist_hit_roll
                     rand = 1..100
                 [/set_variable]
-                [store_unit_defense]
+                [store_unit_defense_on]
                     find_in = opportunist_hit_unit
                     variable = opportunist_hit_chance
-                [/store_unit_defense]
+                [/store_unit_defense_on]
```

Comment:
This needs a corresponding change in the lines that follow. For the deprecated `[store_unit_defense]` higher values are worse, but the replacement has higher values being better. So this is the opportunist_miss_chance.